code/languages/com.mbeddr.mpsutil/languages/com.mbeddr.mpsutil.graphstream/runtime/com.mbeddr.mpsutil.graphstream.runtime.msd

```diff
@@ -57,6 +57,7 @@
     <module reference="5787a8ed-1486-4469-94b0-fa3fc6c8538d(com.mbeddr.mpsutil.graphstream)" version="0" />
     <module reference="ab71436a-a7d1-4689-ac02-b5fde2ec681f(com.mbeddr.mpsutil.graphstream.runtime)" version="0" />
     <module reference="c0488c1e-322f-4f38-92d4-5520a7ce96c1(com.mbeddr.mpsutil.plantuml.pluginSolution)" version="0" />
+    <module reference="563a0770-eb88-4f4f-83ed-f708776fc2fe(de.itemis.stubs.batik)" version="0" />
     <module reference="5b1f863d-65a0-41a6-a801-33896be24202(jetbrains.mps.ide.editor)" version="0" />
     <module reference="ceab5195-25ea-4f22-9b92-103b95ca8c0c(jetbrains.mps.lang.core)" version="0" />
     <module reference="5ad14eca-28d7-4bce-b8e0-648908a49062(org.apache.batik)" version="0" />
```

Comment:
Although the actual solution is removed now, there is still this version reference, which is weird. Maybe check if everything has been properly force-saved?

Reply:
Removed. It was part of the depdenecy version..